<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>827</width>
    <height>618</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Widget">
   <property name="minimumSize">
    <size>
     <width>827</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLWidget" name="glwidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Configuration">
      <item>
       <widget class="QGroupBox" name="TreeOptions">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Options</string>
        </property>
        <widget class="QGroupBox" name="RedSliders">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>171</width>
           <height>151</height>
          </rect>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>204</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>204</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>145</red>
               <green>141</green>
               <blue>126</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="title">
          <string>Red Sliders</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Max Slider</string>
          </property>
         </widget>
         <widget class="QSlider" name="MaxSlider_R">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="MaxValue_R">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.7</string>
          </property>
         </widget>
         <widget class="QSlider" name="MinSlider_R">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="sliderPosition">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Min Slider</string>
          </property>
         </widget>
         <widget class="QLabel" name="MinValue_R">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="GreenSliders">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>171</width>
           <height>151</height>
          </rect>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>154</green>
               <blue>6</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>78</red>
               <green>154</green>
               <blue>6</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Highlight">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>145</red>
               <green>141</green>
               <blue>126</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="title">
          <string>Green Sliders</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Max Slider</string>
          </property>
         </widget>
         <widget class="QSlider" name="MaxSlider_G">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="MaxValue_G">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.7</string>
          </property>
         </widget>
         <widget class="QSlider" name="MinSlider_G">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="sliderPosition">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Min Slider</string>
          </property>
         </widget>
         <widget class="QLabel" name="MinValue_G">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="BlueSliders">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>171</width>
           <height>151</height>
          </rect>
         </property>
         <property name="title">
          <string>Blue Sliders</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Max Slider</string>
          </property>
         </widget>
         <widget class="QSlider" name="MaxSlider_B">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>110</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="sliderPosition">
           <number>70</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="MaxValue_B">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.7</string>
          </property>
         </widget>
         <widget class="QSlider" name="MinSlider_B">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>160</width>
            <height>16</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="sliderPosition">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Min Slider</string>
          </property>
         </widget>
         <widget class="QLabel" name="MinValue_B">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>54</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>0.3</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>510</y>
           <width>181</width>
           <height>80</height>
          </rect>
         </property>
         <property name="title">
          <string>Light Position</string>
         </property>
         <widget class="QDoubleSpinBox" name="x_light">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>-2.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="z_light">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>-2.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="y_light">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="minimum">
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>3.000000000000000</double>
          </property>
         </widget>
        </widget>
        <zorder>groupBox</zorder>
        <zorder>RedSliders</zorder>
        <zorder>GreenSliders</zorder>
        <zorder>BlueSliders</zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>828</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <signal>SetFaces(QString)</signal>
    <signal>SetVertices(QString)</signal>
    <signal>SetFramerate(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>updated_plane(double,double,double,double,bool)</signal>
  <slot>toggle_transfer_function_dialog()</slot>
 </slots>
</ui>
